controllers/feeds/getOneFeedChat.js

```diff
@@ -69,7 +70,7 @@ const getOneFeedChatService = async (req, res) => {
     if (moment(data.results[0].expired_at).isBefore(moment().utc())) {
       return ErrorResponse.e400(res, 'Feed alredy expired');
     }
-    const {comments, upvotes, downvotes} = getReaction(
+    let {comments, upvotes, downvotes} = getReaction(
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `const` to `let` is appropriate here as the variables `comments`, `upvotes`, and `downvotes` are reassigned later in the code.



<!-- This is an auto-generated comment by OSS CodeRabbit -->